custom_components/climate_group/climate.py

```diff
@@ -163,13 +151,26 @@ def should_poll(self) -> bool:
                 climate.DOMAIN, climate.SERVICE_SET_TEMPERATURE, data, blocking=True)
 
     async def async_set_operation_mode(self, operation_mode):
-        """Forward the turn_on command to all climate in the climate group. LEGACY CALL. This will be used only if the hass version is old."""
+        """Forward the turn_on command to all climate in the climate group. LEGACY CALL.
+        This will be used only if the hass version is old."""
         data = {ATTR_ENTITY_ID: self._entity_ids,
                 ATTR_HVAC_MODE: operation_mode}
 
         await self.hass.services.async_call(
             climate.DOMAIN, climate.SERVICE_SET_HVAC_MODE, data, blocking=True)
         
+    @property
+    def preset_mode(self):
+        """Return the current preset mode, e.g., home, away, temp."""
+        if self._is_away:
```

Comment:
why only support `PRESET_AWAY` while we could support all preset modes? (see comment below)

Reply:
honestly, because my 'stats only support `PRESET_AWAY`, and I haven't checked what other options there are 🤷‍♂

Reply:
fixed as above